src/deprecated.js

```diff
@@ -38,5 +39,6 @@ jQuery.isArray = Array.isArray;
 jQuery.parseJSON = JSON.parse;
 jQuery.nodeName = nodeName;
 jQuery.isWindow = isWindow;
+jQuery.cssProps = cssProps;
```

Comment:
I'm afraid it might be a breaking change. The fact that the object is now injected instead of used directly from `jQuery.cssProps` means that if someone overwrites the object completely it will stop working.

Perhaps we should just move defining this object here and in source, where it's used, make sure code still works fine if it's not defined.

Or just deprecate in docs without touching the code base. Whatever you prefer.

Reply:
The same for an API ticket.

Reply:
Good point. Maybe there's nothing to be done here. cssProps brought up no search results on api.jquery.com, so I'm not sure it's actually mentioned anywhere. Maybe there's nothing to do here?